app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java

```diff
@@ -72,6 +96,11 @@ public View getChildView(int groupPosition, final int childPosition,
         return convertView;
     }
 
+    /**
+     * returns total children count.
```

Comment:
@top-gun007  what @sandarumk necessarily meant was there is no specific need to commit after you make changes to every file. What she effectively meant was to make meaningful commits. 
And yes, by my reckoning, there are just a little too many commits for this particular PR, please squash them. Maybe @sandarumk has something to add..